docs/guide/faq.md

```diff
@@ -0,0 +1,11 @@
+# Frequently Asked Questions
+
+## What is the difference between tsdown and Rolldown ?
+
+## Why should I use tsdown instead of other bundlers (like tsup, unbuild,...) ?
+
+## Will tsdown support stub mode (similar to unbuild) ?
+
+For various technical and design reasons, we have decided not to support stub mode in tsdown. We recommand using the [watch mode](/guide/watch-mode.md) instead.
```

Comment:
```suggestion
For [various technical and design reasons](https://github.com/rolldown/tsdown/pull/164#issuecomment-2849720617), we have decided not to support stub mode in tsdown. We recommend using the [watch mode](/guide/watch-mode.md) instead.
```

Also might be worth to link to the issue at "various technical reasons"

Reply:
I think duplicated links are confusing. I'm going to improve the expression.